### ISSUE TITLE
YAM crashes when hitting "clear warning" under TCP/IP

### ISSUE THREAD

**Issue by jens-maus** [jens-maus/yam#605]
**:bust_in_silhouette: Originally by _christoph.poelzl@rmsvideo.ch_ on 2015-08-20 10:16:17 +0200**
<hr>
## Summary

when I hit "clear warnings" under TCP/IP I get a crash. Crash log attached. This is with MorphOS version of YAM.

## Steps to reproduce
 1.
 2.

## Notes

**Comment by jens-maus** [comment]
**:bust_in_silhouette: Originally by _christoph.poelzl@rmsvideo.ch_ on 2015-08-20 10:19:32 +0200**
<hr>
Ah, I did get yesterday the following warning: please see attached sg.

**Comment by jens-maus** [comment]
**:bust_in_silhouette: Originally by @tboeckel on 2015-08-20 22:42:25 +0200**
<hr>
In "8333" 8333:
```
#CommitTicketReference repository="" revision="8333"
* mui/TCPIPConfigPage.c: added a missing parameter for a method invocation when clearing the SSL certificate warnings. This closes #605.
```
